Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/Tags.md

```diff
@@ -0,0 +1,43 @@
+# Tags
+`Alias: Umbraco.Tags`
+
+`Returns: CSV` or `JSON`
+
+The Tags property editor allows you to add multiple tags to a node.
+
+## Data Type Definition Example
+
+![Data Type Definition Example](images/tags/configuration.png)
+
+Data can be saved in either CSV format or in JSON format. By default data is saved in CSV format. The difference between using CSV and JSON is that with JSON you can save a tag, which includes comma seperated values.
+
+Since the release of Umbraco 7.6 there are built-in property value converters, which means you don't need to worry about writing them yourself or parse the JSON output when choosing "JSON" in the storage type field. Therefore code example last on this page will work out of the box without further ado.
```

Comment:
last sentence sounds a bit funny. Wat is "last"? Might be my limited knowledge of English.